src/pages/video-test.tsx

```diff
@@ -89,6 +88,13 @@ const VideoTest: React.FC<any> = ({videoResource}) => {
             label="English"
             default
           />
+          <track
+            id="notes"
+            src="https://gist.githubusercontent.com/joelhooks/bd3c1d68cb5a67adfcd6c035200d1fde/raw/aa7060f584e04db26c5fa6b464bf2058ed6f6e93/notes.vtt"
+            kind="metadata"
+            label="notes"
+          />
+          <CueBar order={6.0} />
```

Comment:
these changes are only available on `/video-test` route so this is safe to merge